daemon/info.go

```diff
@@ -142,9 +142,17 @@ func (daemon *Daemon) showPluginsInfo() types.PluginsInfo {
 		pluginsInfo.Network = append(pluginsInfo.Network, nd)
 	}
 
+	pluginsInfo.Authorization = daemon.GetAuthorizationPluginsList()
+
 	return pluginsInfo
 }
 
+// GetAuthorizationPluginsList returns the list of plugins drivers
+// registered for authorization.
+func (daemon *Daemon) GetAuthorizationPluginsList() []string {
```

Comment:
I'd remove this getter as everything from Config Store is accessed directly in daemon package (note labels above for instance)